src/fs-plus.coffee

```diff
@@ -287,7 +283,7 @@ fsPlus =
   # Public: Create a directory at the specified path including any missing
   # parent directories asynchronously.
   makeTree: (directoryPath, callback) ->
-    fs.exists directoryPath, (exists) ->
+    fsPlus.isDirectory directoryPath, (exists) ->
```

Comment:
@kevinsawicki The tests are passing, however, I see here a little thing: what happens if the path is a file?